### PR TITLE
Update with minimatch pattern for excluding files by extension

### PR DESCRIPTION
**Pull request by kzu**
Also link to the fnmatch docs for the format.

See https://github.com/microsoft/azure-pipelines-tasks/issues/4936#issuecomment-497026569

### CODE REVIEW

**Comment by PRMerger10** [comment]
@kzu : Thanks for your contribution! The author, @vtbassmatt, has been notified to review your proposed change.

File: docs/pipelines/tasks/utility/delete-files.md [hunk 1]
```diff
@@ -56,12 +56,15 @@ None
...
 <li>```**\temp*``` deletes any file or folder with a name that begins with temp.</li>
 <li>```**\temp*\**``` deletes files in any sub-folder that begins with the name temp.</li>
+<li>```!(*.vsix)``` deletes all  in the root folder that do not have a `.vsix` extension.</li>
 </ul>
 </td>
 </tr>
 [!INCLUDE [temp](../_shared/control-options-arguments.md)]
 </table>
 
+https://github.com/microsoft/azure-pipelines-tasks/issues/4936#issuecomment-497026569
```
Comment (vtbassmatt) [inline docs/pipelines/tasks/utility/delete-files.md]: Please remove this, and then I'll merge.

**Comment by PRMerger12** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
We resolved the merge conflict and made the change that @vtbassmatt requested.

**Pull request merged**